src/pages/workspace/members/WorkspaceMemberDetailsPage.tsx

```diff
@@ -80,6 +82,7 @@ function WorkspaceMemberDetailsPage({personalDetails, policy, route}: WorkspaceM
     const ownerDetails = personalDetails?.[policy?.ownerAccountID ?? -1] ?? ({} as PersonalDetails);
     const policyOwnerDisplayName = ownerDetails.displayName ?? policy?.owner ?? '';
     const hasMultipleFeeds = Object.values(cardFeeds?.settings?.companyCards ?? {}).filter((feed) => !feed.pending).length > 0;
+    const paymentAccountID = cardSettings?.paymentBankAccountID ?? 0;
```

Comment:
Nab just to make it clearer
```suggestion
    const paymentBankAccountID = cardSettings?.paymentBankAccountID ?? 0;
```